src/Sylius/Bundle/ThemeBundle/Filesystem/FilesystemInterface.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\ThemeBundle\Filesystem;
+
+use Symfony\Component\Filesystem\Exception\FileNotFoundException;
+use Symfony\Component\Filesystem\Exception\IOException;
+
+/**
+ * @author Kamil Kokot <kamil.kokot@lakion.com>
```

Comment:
I believe there is some copy paste here from Symfony, let's add original authors too. :)